Core/src/org/sleuthkit/autopsy/modules/hashdatabase/HashDbManager.java

```diff
@@ -786,48 +1031,340 @@ public boolean equals(Object obj) {
+                orgName = Bundle.HashDbManager_CentralRepoHashDb_orgError();
+            }
+        }
+
+        /**
+         * Adds a listener for the events defined in HashDb.Event.
+         * Listeners are used during indexing.
+         *
+         * @param pcl
+         */
+        @Override
+        public void addPropertyChangeListener(PropertyChangeListener pcl) {
+            propertyChangeSupport.addPropertyChangeListener(pcl);
+        }
+
+        /**
+         * Removes a listener for the events defined in HashDb.Event.
+         *
+         * @param pcl
+         */
+        @Override
+        public void removePropertyChangeListener(PropertyChangeListener pcl) {
+            propertyChangeSupport.removePropertyChangeListener(pcl);
+        }
+        
+        @Override
+        public boolean hasIndexOnly() throws TskCoreException{
+            return true;
+        }
+
+        @Override
+        public String getHashSetName() {
+            return hashSetName;
+        }
+        
+        @Override
+        String getDisplayName(){
+            return getHashSetName() + " " + getVersion();
+        }
+        
+        String getVersion(){
+            return version;
+        }
+        
+        String getOrgName(){
+            return orgName;
+        }
+        
+        int getReferenceSetID(){
+            return referenceSetID;
+        }
+
+        @Override
+        public String getDatabasePath() throws TskCoreException {
+            return "";
+        }
+
+        @Override
+        public String getIndexPath() throws TskCoreException {
+            return "";
+        }
+
+        @Override
+        public SleuthkitHashSet.KnownFilesType getKnownFilesType() {
+            return knownFilesType;
+        }
+
+        @Override
+        public boolean getSearchDuringIngest() {
+            return searchDuringIngest;
+        }
+        
+        @Override
+        boolean getDefaultSearchDuringIngest(){
+            // Central repo hash sets are off by default
+            return false;
+        }
+
+        @Override
+        void setSearchDuringIngest(boolean useForIngest) {
+            this.searchDuringIngest = useForIngest;
+        }
+
+        @Override
+        public boolean getSendIngestMessages() {
+            return sendIngestMessages;
+        }
+
+        @Override
+        void setSendIngestMessages(boolean showInboxMessages) {
+            this.sendIngestMessages = showInboxMessages;
+        }
+
+        /**
+         * Indicates whether the hash database accepts updates.
+         *
+         * @return True if the database accepts updates, false otherwise.
+         *
+         * @throws org.sleuthkit.datamodel.TskCoreException
+         */
+        @Override
+        public boolean isUpdateable() throws TskCoreException {
+            return false;
+            // TEMP this will change as soon as adding to the database is supported 
+            // return (! readOnly);
```

Comment:
Coming in one of the other PRs?

Reply:
Yes the one that allows the user to add hashes to a central repo db